crates/console-timer/Cargo.toml

```diff
@@ -0,0 +1,14 @@
+[package]
+name = "gloo-console-timer"
+version = "0.1.0"
+authors = ["Christian Brevik <christian.brevik@gmail.com>"]
```

Comment:
Can you make this `"The Rust and WebAssembly Working Group"` like the root `Cargo.toml` does?

I just realized that the `crates/timers/Cargo.toml` doesn't use that same author line, and I'll file a PR to fix it.

Reply:
Woops, didn't check the root `Cargo.toml`. Will fix!